builtin/logical/ssh/path_roles.go

```diff
@@ -672,6 +684,11 @@ func (b *backend) checkUpgrade(ctx context.Context, s logical.Storage, n string,
 		err = nil
 	}
 
+	if result.Version < 3 {
+		modified = true
+		result.NotBeforeDuration = 30 * time.Second
```

Comment:
One last tiny nit, sorry: this needs a `result.Version = 3` so we don't keep overwriting the roles :-)

This is done in the conditional so that all-at-once updates from older versions of Vault work (incrementally applying new versions of the role from the 0-valued ones). If we don't set it here, I think this will run every load of the role, potentially overwriting the specified value. 

```suggestion
		result.NotBeforeDuration = 30 * time.Second
		result.Version = 3
```

Reply:
Eh, or not :D Still, no point holding up the ACK on that.